src/bridge/MatrixHandler.ts

```diff
@@ -367,9 +395,13 @@ export class MatrixHandler {
         });
 
         // Check if this room is known to us.
-        const hasExistingRoom = (
-            await this.ircBridge.getStore().getIrcChannelsForRoomId(event.room_id)
-        ).length > 0;
+        const rooms = await this.ircBridge.getStore().getIrcChannelsForRoomId(event.room_id);
+        const hasExistingRoom= rooms.length > 1;
+
+        const inviteeIsVirtual = !!this.ircBridge.getServerForUserId(event.state_key);
+        const inviterIsVirtual = !!this.ircBridge.getServerForUserId(event.sender);
+
+        console.log(rooms, event.sender)
```

Comment:
Is `console.log` allowed in production or is this a leftover from debugging?

Reply:
leftover :man_facepalming: